core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/details/NFCDocumentTag.kt

```diff
@@ -137,6 +137,36 @@ class NFCDocumentTag(val readDG2: Boolean = true) {
                     additionalPersonDetails.title = dg11.title
 
                     passport.additionalPersonDetails = additionalPersonDetails
+
+                    // Hash Checking
+                    val hashCheckNotSucceeded = "hash-check not SUCCEEDED"
+                    if (verifySecurity.ht != VerificationStatus.Verdict.SUCCEEDED) {
+                        Log.e(TAG, hashCheckNotSucceeded)
+                    }
+
+                    // when enabled, send logs to Sentry
+                    if (captureLog) {
+                        // Send to Sentry hashCheckNotSucceeded
+                        Sentry.captureMessage(hashCheckNotSucceeded)
+
+                        // DG11 NameOfHolder
+                        val dg11NameOfHolder : String? = dg11.nameOfHolder ?: null
+                        if (dg11NameOfHolder == null) {
+                            // Send to sentry null nameOfHolder
+                            Sentry.captureMessage( "nameOfHolder = $dg11NameOfHolder, len = ${dg11NameOfHolder?.length}")
+                        } else {
+                            // Log nameOfHolder in app
+                            Log.i(TAG, dg11NameOfHolder)
```

Comment:
I am curious to know the `len` of `dg11NameOfHolder` such that perhaps it may indeed appear empty, but it has some length in here. Need to log to sentry the `len` value. If it is null, then just log to sentry that dg11NameOfHolder is null.

Reply:
Noted. will update this @typelogic

Reply:
@typelogic already updated. Please see 04edc9c.